src/commands/createNewExercise.ts

```diff
@@ -39,6 +39,7 @@ export async function createNewExercise(exercise: Exercise) {
 
         // File already exists, but it is a different exercise. Ask the user to
         // provide a different filename.
+        window.showWarningMessage("Choose a different filename. File already exists, but it's a different exercise.");
```

Comment:
```suggestion
        window.showWarningMessage("A file with the same name already exists on your system for a different exercise already exists. Please choose another name.");
```

Reply:
I chose to make it short and place the instruction first, because most users won't expand the notification. This results in losing the rest of the sentence. Do you have another suggestion?
![image](https://user-images.githubusercontent.com/56451049/151345907-1d132862-5c57-4e5f-9689-f49c6086e3ba.png)

Reply:
I'd maybe suggest something like "This filename is already in use for a different exercise..." which contains the most important info before being cut off by the notification, and is ~15 characters shorter than what is cut off in the screenshot so it should fit on the notification.

Extra details can be added in the rest of the sentence, which can be cut off without losing any information. If it's followed up by, for example, "... choose a different name for this exercise." then it should still display "choose a diff..." which I feel should be clear enough to the users.